modules/ROOT/pages/auto-updates.adoc

```diff
@@ -42,4 +40,24 @@ The following finalization strategies are available:
  * immediately reboot to apply an update, as soon as it is downloaded and staged locally
  * use an external lock-manager to reboot a fleet of machines in a coordinated way
 
-For further documentation on how to configure updates finalization, check Zincati documentation on https://github.com/coreos/zincati/blob/0.0.6/docs/usage/updates-strategy.md[updates strategy].
+A specific finalization strategy can be configured on each node.
+
+The https://github.com/coreos/fcct[FCCT] snippet below shows how to use a custom lock-manager:
+
+.Example: configuring Zincati updates strategy
+[source,yaml]
+----
+variant: fcos
+version: 1.0.0
+storage:
+  files:
+    - path: /etc/zincati/config.d/55-updates-strategy.toml
+      contents:
+        inline: |
+          [updates]
+          strategy = "fleet_lock"
+          [updates.fleet_lock]
+          base_url = "https://example.com/lock-manager/"
+----
+
+For further details on updates finalization, check https://github.com/coreos/zincati/blob/0.0.6/docs/usage/updates-strategy.md[Zincati documentation].
```

Comment:
Maybe, check *the* Zincati documentation?

Reply:
Ack.